imports/plugins/core/orders/client/templates/workflow/cancelOrder.html

```diff
@@ -0,0 +1,41 @@
+<template name="coreOrderCancelOrder">
+  <div aria-labelledby="coreOrderCancelOrder" class="panel panel-default" id="coreOrderCancelOrder-panel" role="tabpanel">
+     <div class="panel-heading">
+      <h3 class="panel-title">Cancel User's Order</h3>
+     </div>
+     {{#if showCancelOrderForm}}
+     <div class="panel-body">
+      <form name="cancelOrderForm">
+        <div class="form-group {{hasError messages.errors.comment}}">
```

Comment:
I think `{{hasError messages.errors}}` is sufficient, same below
Why? - if there are no errors messages.errors would be undefined
This simplifies your hasError helper below

Maybe also rename helper to getErrorClass, cause that is basically what it does